src/components/FilterSearch.tsx

```diff
@@ -115,13 +115,15 @@ export function FilterSearch({
       if (currentFilter) {
         searchActions.setFilterOption({ ...currentFilter, selected: false });
       }
-      searchActions.setFilterOption({ ...newFilter, displayName: newDisplayName, selected: true });
-      setCurrentFilter(newFilter);
-      setFilterQuery(value);
-      if (select && searchOnSelect) {
-        searchActions.setOffset(0);
-        searchActions.resetFacets();
-        executeSearch(searchActions);
+      if (select) {
```

Comment:
`select` is always true right? I might be missing something but I think `handleDropdownEvent` is only called within `handleSelectDropdown` which always has the 3rd param as true. Could we move `handleDropdownEvent` into `handleSelectDropdown`? There's no need to have an extra wrapper function

Reply:
Probably we used to have `handleDropdownEvent` called in 2+ places and we removed the others

Reply:
`handleDropdownEvent` is also called by `handleToggleDropdown`

Reply:
I think we want to refactor things to remove this `select` boolean but we can procrastinate on that! It's normally not very readable to have a boolean flag to "do something"